### DISCUSSION TITLE
Apexcharts graph inline script issue

### DISCUSSION THREAD

**Discussion by devster31** [marp-team/marp#251] [Q&A]
I'm trying to use [Apexcharts.js](https://apexcharts.com/) as an inline script, like so:
```md
---
marp: true
---

<script src="https://cdn.jsdelivr.net/npm/apexcharts@3/dist/apexcharts.min.js" unsafe-inline></script>

<script> my apex chart </script>
```
in a slide.
The graphical output is OK except for the annotations, which aren't positioned appropriately, and the cursor position which is not correct (graph highlight is not below mouse, but to the left or the right).
I'm trying to understand if there's a way to fix this or if this is a side-effect of the final output (meaning the javascript used to turn slides is interfering with the graph one).
Any insight?

**Comment by yhatt** [comment]
In HTML output, each Marp slide will scale to fit the window automatically.

But Apexcharts is not respecting the scale applied by CSS `transform` when getting mouse position for tooltip. (Marp's slide scaling is using SVG transformation instead of CSS but the same can be said)

Thus, Apexcharts mouse tracking behaves as if the slide was not scaled. One of Apexcharts contributors makes clear this is a limitation of the library.
https://github.com/apexcharts/apexcharts.js/issues/1849

**Reply by devster31** [reply]
Is there a way to disable automatic scaling for `marp`?

**Reply by yhatt** [reply]
No. Technically it may be possible to disable by removing `viewBox` attribute from each `<svg>`s via Marp plugin, but the output of HTML will be mess the slide deck up.
